pkg/fanal/analyzer/language/python/packaging/packaging.go

```diff
@@ -98,12 +194,13 @@ func (a packagingAnalyzer) open(file *zip.File) (dio.ReadSeekerAt, error) {
 }
 
 func (a packagingAnalyzer) Required(filePath string, _ os.FileInfo) bool {
-	for _, r := range requiredFiles {
-		if strings.HasSuffix(filePath, r) {
-			return true
-		}
-	}
-	return false
+	return strings.Contains(filePath, ".dist-info") || required(filePath)
+}
+
+func required(filePath string) bool {
```

Comment:
Can you add a comment on what files this function finds please?

Reply:
I renamed the function